src/graphql/resolvers/stockResolver.js

```diff
@@ -1,5 +1,6 @@
 const Market = require('../../models/market');
 const Stock = require('../../models/stock');
+const PubSub = require('../subscriptions');
```

Comment:
Change the name from PubSub to something more meaningful. Maybe just keep it subscriptions.